demo/components/Calendar/Localization.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import * as L from '../../../leda';
+import { globalDefaultMessages } from '../../../leda/components/LedaProvider/globalDefaultMessages';
+
+const customMessages = {
+  calendar: {
+    monthNames: ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio', 'Julio', 'Agosto', 'Septiembre', 'Octubre', 'Noviembre', 'Deciembre'],
+    shortMonthNames: ['Ene', 'Feb', 'Mar', 'Abr', 'May', 'Jun', 'Jul', 'Ago', 'Sep', 'Oct', 'Nov', 'Dec'],
+    weekDays: ['Domingo', 'Lunes', 'Martes', 'Miercoles', 'Jueves', 'Viernes', 'Sabado'],
+    shortWeekDays: ['Dom', 'Lun', 'Mar', 'Mie', 'Jue', 'Vie', 'Sab'],
+    firstWeekDay: 0,
+  }
+} satisfies typeof globalDefaultMessages
+
+export const Localization = (props: any) => {
+  const [value, setValue] = React.useState<Date>(new Date);
+
+  return (
+    <L.Leda messages={customMessages}>
```

Comment:
I don't think messages is a good choice here, it is rather labels or texts or something else

https://en.wikipedia.org/wiki/Message

Or it can be named as localisation if it is intended to be a universal interface for all texts 

And it would be really nice to have an exported interface to help people avoid mistakes

Reply:
Oh, weird, we always named these messages because of the react-intl terminology

Reply:
So usually they are called messages
But I'm ok with any name)

Reply:
Cool, I've never used that, so let it be messages

Anyway, to me a message is something I'm typing right now, not a Mon or Tue 😑